src/home/form/LoginForm.jsx

```diff
@@ -32,9 +33,9 @@ class Loginpage extends React.Component {
     }
 
     sendToServer(e) {
-        console.log('LoginForm is submitting the DATA MAN');
         e.preventDefault();
         var self = this;
+        this.setState({logInTime:Date.now()});
         axios.post('/login', this.state)
```

Comment:
We shouldn't be doing a set state in this method, it's async and would cause the component to re-render. Better to just extract our the user details to a separate object and post that.